client/run.go

```diff
@@ -290,6 +303,66 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 	return string(jsonString), nil
 }
 
+func parseVolume(volStr string) (*pod.UserVolume, *pod.UserVolumeReference, error) {
+
+	var (
+		vol      = pod.UserVolume{}
+		volRef   = pod.UserVolumeReference{}
+		srcName  string
+		destPath string
+		readOnly string
+	)
+
+	fields := strings.Split(volStr, ":")
+	if len(fields) == 3 {
+		// cmd: -v host-src:container-dest:rw
+		srcName = fields[0]
+		destPath = fields[1]
+		readOnly = fields[2]
+		if readOnly != "ro" && readOnly != "rw" {
+			return nil, nil, fmt.Errorf("flag only support(ro or rw): --volume")
+		}
+	} else if len(fields) == 2 {
+		// cmd: -v host-src:container-dest
+		srcName = fields[0]
+		destPath = fields[1]
+		readOnly = "rw"
+	} else if len(fields) == 1 {
+		// -v container-dest
+		srcName = ""
```

Comment:
in golang, "" is the initial/default value of a string. please remove this line.